bin/configure-monitoring-secret.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# -----------------------------------------------------------------------------
+# This script creates SSM parameter for storing integration URL for incident management
+# services. Script creates new SSM attribute or updates existing. 
+#
+# Positional parameters:
+#   APP_NAME (required) – the name of subdirectory of /infra that holds the
+#     application's infrastructure code.
+#   ENVIRONMENT is the name of the application environment (e.g. dev, staging, prod)
+#   INTEGRATION_ENDPOINT_URL is the url for the integration endpoint for external 
+#   incident management services (e.g. Pagerduty, Splunk-On-Call)
+# -----------------------------------------------------------------------------
+set -euo pipefail
+
+APP_NAME=$1
+ENVIRONMENT=$2
+INTEGRATION_ENDPOINT_URL=$3
+
+terraform -chdir=infra/$APP_NAME/app-config init > /dev/null
+terraform -chdir=infra/$APP_NAME/app-config refresh > /dev/null
+
+HAS_INCIDENT_MANAGEMENT_SERVICE=$(terraform -chdir=infra/$APP_NAME/app-config output -raw has_incident_management_service)
+if [ $HAS_INCIDENT_MANAGEMENT_SERVICE = "false" ]; then
+  echo "Application does not have incident management service, no secret to create"
+  exit 0
+fi
```

Comment:
in case has_incident_management_service set to false env_config output returns null instead of secret name, so with current logic I still need to ensure app_config => has_incident_management_service is set to true

Reply:
ah ok makes sense thanks